src/components/pagination/pageControls/pageControls.module.scss

```diff
@@ -12,4 +14,5 @@
   text-overflow: ellipsis;
   white-space: nowrap;
   text-align: left;
+  @include font-scale();
```

Comment:
May be it make sense to change the `font-scale` usage once in the `pagination.module.scss`?
In this case most of the components will inherit the font values.